src/main/java/com/mixpanel/android/mpmetrics/AnalyticsMessages.java

```diff
@@ -8,6 +8,7 @@
 import android.os.Process;
 import android.os.SystemClock;
 import android.support.annotation.NonNull;
+import android.util.Log;
```

Comment:
Looks like it's unused?

Reply:
yea.. relied on AS to take care of unused imports🤷🏻‍♂️